README.md

```diff
@@ -71,19 +71,20 @@ You can read a complete set of standardised definitions in the [Solid dictionary
 # Solid Conversations
 The  Gitter chats associated to the [Solid GitHub account](https://github.com/solid) include 
 
-* solid/chat: a place to talk about all things Solid
-* solid/solid-spec: a place to talk about the Solid specification. Note that actual changes to the Solid specification happen through the official processes. 
+* [solid/chat](https://gitter.im/solid/chat): a place to talk about all things Solid
+* [solid/solid-spec](https://gitter.im/solid/solid-spec): a place to talk about the Solid specification. Note that actual changes to the Solid specification happen through the official processes. 
 
 Groups implementing the Solid  have a dedicated channel based on the implemtatnion. 
-* solid/app-development: Solid app developers can share tips and ask questions on this channel 
-* solid/pod-providers: coming soon
-* solid/events: coming soon
+* [solid/app-development](https://gitter.im/solid/app-development): Solid app developers can share tips and ask questions on this channel 
+* [solid/pods](https://gitter.im/solid/pods): coming soon
+* [solid/events](https://gitter.im/solid/solid-events): coming soon
 
 Solid Projects have dedicated chat channels. 
-* solid/node-solid-server: originally a place to talk about [Solid Project NSS-v5.0.0](https://github.com/orgs/solid/projects/1) and now a dedicated space for [Solid Project ASAP on Server](https://github.com/orgs/solid/projects/2)
-* solid-panes: coming soon
-* site-and-docs: coming soon
+* [site-and-docs](https://gitter.im/solid/site-and-docs): a place to talk about the Solid Project '[Solid Content Generation](https://github.com/orgs/solid/projects/6)'
```

Comment:
might want to changes [site-and-docs] to [solid/site-and-docs] for the sake of consistency?